litgpt/data/openwebtext.py

```diff
@@ -68,14 +68,14 @@ def tokenize(data: Dataset, index: int):
             fn=partial(tokenize, split_dataset["train"]),
             inputs=list(range(len(split_dataset["train"]))),
             output_dir=self.data_path_train,
-            num_workers=(os.cpu_count() - 1),
+            num_workers=min(64, os.cpu_count() - 1),
             chunk_bytes="200MB",
         )
         optimize(
             fn=partial(tokenize, split_dataset["val"]),
             inputs=list(range(len(split_dataset["val"]))),
             output_dir=self.data_path_val,
-            num_workers=(os.cpu_count() - 1),
+            num_workers=min(8, os.cpu_count() - 1),
```

Comment:
That's a good change. There were a couple of people who reported issues where everything hangs / gets very slow, and it might have been due to too many workers and processes.